db/blip_handler.go

```diff
@@ -380,6 +383,11 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 	}()
 
 	base.InfofCtx(bh.loggingCtx, base.KeySync, "Sending changes since %v", opts.since)
+	collectionCtx, err := bh.collections.get(bh.collectionIdx)
+	if err != nil {
+		base.WarnfCtx(bh.loggingCtx, "[%s] Could not get collection in sendChanges: %s", err)
+		return
+	}
```

Comment:
We've already done this (or have the `blipSyncCollectionContext` available) in the callers of `sendChanges`, so maybe we store `collectionCtx.changesCtx` in `sendChangesOptions` given `bh.collections.get` would be doing locking/map lookup work twice in most cases for this.